### PR TITLE
Migrate website from html with bootstarp

### PR DESCRIPTION
**Pull request by sebatyler**
Current website is https://starrynight.world/ and made by html with bootstrap.
I tried to migrate from it to flutter.

### CODE REVIEW

**Review by hayeonhee — APPROVED** [review]
Looks good to me. 😄

**Comment by sebatyler** [comment]
Sorry that I pushed a new commit. 😅 Could you review that again?

**Comment by sebatyler** [comment]
I've done to set up running `flutter test` automatically.

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: web
```
Comment (sebatyler) [inline .github/workflows/test.yaml:1]: This file is for running `flutter test` in github actions.
Configuration is so simple 😄

**Comment by sebatyler** [comment]
Thank you for the review approval.

**Pull request merged**